### DISCUSSION TITLE
Unable to pull images from ghcr

### DISCUSSION THREAD

**Discussion by Pothulapati** [community/community#26889] [Actions]
<p>We have images pushed into ghcr as public, but the pulls fail stating</p>
<pre><code class="lang-auto"> docker pull ghcr.io/linkerd/proxy-init:v1.3.6
v1.3.6: Pulling from linkerd/proxy-init
afb6ec6fdc1c: Already exists
80cf89d7fa2c: Pulling fs layer
145a4ea92b67: Pulling fs layer
a22f2d4faea9: Pulling fs layer
error pulling image configuration: unknown blob
</code></pre>
<p>The image can be viewed at <a href="https://github.com/orgs/linkerd/packages/container/package/proxy-init" rel="noopener nofollow ugc">https://github.com/orgs/linkerd/packages/container/package/proxy-init</a></p>
<p>Is ghcr down? If yes, I would expect Github Status to show the same.</p>
<p>Thanks</p>
<p><strong>Update</strong>: Pulls work now, but Was it down?, as other users also expected the same, Can you please make sure that the Github Status is updated in real-time?</p>
<p>Thanks</p>

**Comment by clarkbw** [comment, marked as answer]
<p>Fixed now. Had an issue this morning. We’ll review status reporting to see if this should have triggered something. Thanks.</p>
